test/statetest/statetest_loader.cpp

```diff
@@ -11,11 +12,6 @@ namespace evmone::test
 namespace json = nlohmann;
 using evmc::from_hex;
 
-namespace
```

Comment:
Shouldn't these stay in the anonymous namespace?

Reply:
No, because `from_json` is now also defined in header.